spec/requests/product_drives_requests_spec.rb

```diff
@@ -1,21 +1,97 @@
 require 'rails_helper'
 
-RSpec.describe "ProductDrives", type: :request do
-  let(:default_params) do
-    { organization_id: @organization.id.to_param }
+RSpec.describe "ProductDrives", type: :request, skip_seed: true do
+  let(:organization) { FactoryBot.create(:organization) }
+  let(:user) { FactoryBot.create(:user, organization: organization) }
+  let(:default_params) { { organization_id: organization.to_param } }
+
+  context "while not signed in" do
+    it "is unsuccessful" do
+      get product_drives_path(default_params)
+
+      expect(response).not_to be_successful
+    end
   end
 
   context "While signed in >" do
-    let(:product_drive) { create(:product_drive) }
+    let(:product_drive) { create(:product_drive, organization: organization) }
```

Comment:
Just checking because I'am not sure, is this `product_drive` even being used?

Reply:
It is being used, but only in a few tests - I've now moved it so that it's only defined in the tests that require it 👍